src/anaconda/install.sh

```diff
@@ -11,7 +11,7 @@
 VERSION=${VERSION:-"latest"}
 USERNAME=${USERNAME:-"automatic"}
 UPDATE_RC=${UPDATE_RC:-"true"}
-CONDA_DIR=${CONDA_DIR:-"/usr/local/conda"}
+CONDA_DIR="/opt/conda"
```

Comment:
Changing directory as apt-get installs conda here by default, would it need backcompat? 

Should we link `"/opt/conda"` to `"/usr/local/conda"` ?

(btw CONDA_DIR was never exposed as option)

Reply:
I believe this should be fine